x/stakeibc/keeper/host_zone.go

```diff
@@ -207,8 +207,10 @@ func (k Keeper) DecrementValidatorDelegationChangesInProgress(hostZone *types.Ho
 		return errorsmod.Wrapf(types.ErrValidatorNotFound, "validator %s not found", validatorAddress)
 	}
 	if validator.DelegationChangesInProgress == 0 {
-		return errorsmod.Wrapf(types.ErrInvalidValidatorDelegationUpdates,
-			"cannot decrement the number of delegation updates if the validator has 0 updates in progress")
+		// TODO: Revert after upgrade v14
+		// return errorsmod.Wrapf(types.ErrInvalidValidatorDelegationUpdates,
+		// 	"cannot decrement the number of delegation updates if the validator has 0 updates in progress")
+		return nil
```

Comment:
As I understand it this is the 0 floor. In theory this line should not be reachable, so imo we should emit an event here and log before returning nil, so that we can trace any issues. Then in v14 we revert.

Reply:
good call, added a log